lib/widgets/scaffold.dart

```diff
@@ -163,4 +167,58 @@ class AppScaffold extends StatelessWidget {
       ),
     );
   }
+
+  ///
```

Comment:
There should be no empty lines in dartdoc style:
https://dart.dev/guides/language/effective-dart/documentation

Reply:
Removed empty lines:
https://github.com/student-hub/acs-upb-mobile/pull/250/commits/31536835b8d109bd3ef7b078038b0eafbd696c57#diff-91af2d9745e86244940959ec39c99cca9b8b43c1502042d85f16fcc6a311030bR179-R180